src/item_effects.c

```diff
@@ -29,7 +29,12 @@ bool CustomApplyItemEffect(
       // Return true to signal that we've handled the effect
       return true;
     default:
-      // Return false to use the game's normal effect
+#ifdef COT_RUST
+      // If the Rust runtime of c-of-time is used, ask the Rust implementation to process the effect.
+      return eos_rs_apply_item_effect(user, target, item, is_thrown);
```

Comment:
Oh yeah, that would also make it easier to work with with removing the examples. Maybe it would be a good idea to take a similiar grouping approach as with the `eos-rs` crate for C too: As in have `src/` only for the example/user code and have eg. `lib/` for cot internal C code?

Reply:
`src/cot/` contains internal code (but `lib/` would probably be a better name)

Reply:
https://github.com/tech-ticks/c-of-time/pull/13/commits/1f4ad3408544e112fb7f75b44f1a1deb1342c110

@tech-ticks Is this okay? I'm not much of a C dev :clown_face:

Reply:
LGTM in general, UsernameFodder commented some suggestions below.